core/server_config.go

```diff
@@ -203,6 +224,12 @@ func FlagSet() *pflag.FlagSet {
 	flagSet.StringSlice("http.default.cors.origin", nil, "When set, enables CORS from the specified origins for the on default HTTP interface.")
 	flagSet.String("tls.offload", "", "Whether to enable TLS offloading for incoming connections. If enabled `tls.certheader` must be configured as well.")
 	flagSet.String("tls.certheader", "", "Name of the HTTP header that will contain the client certificate when TLS is offloaded.")
+	// Legacy TLS settings, to be removed in v6:
+	flagSet.String("network.certfile", "", "PEM file containing the server certificate for the gRPC server. "+
+		"Required when `network.enabletls` is `true`.")
+	flagSet.String("network.certkeyfile", "", "PEM file containing the private key of the server certificate. "+
+		"Required when `network.enabletls` is `true`.")
+	flagSet.String("network.truststorefile", "truststore.pem", "PEM file containing the trusted CA certificates for authenticating remote gRPC servers.")
```

Comment:
#1334 added a deprecation warning on these flags that are now lost

Reply:
Parallel PRs that are going to conflict either way, so I'll have to re-add them in this PR or the other (whichever gets merged last).